bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcConfiguration.java

```diff
@@ -28,6 +28,7 @@
  */
 public class BcmcConfiguration extends Configuration {
 
+    // TODO remove this when BcmcComponent starts fetching the public key from the client key
```

Comment:
This is blocked until we can fetch the public key in the `BcmcComponent`

Reply:
Looking at it again, I would say maybe just add this to this PR. Move the method from the CardDelegate to somewhere the BCMC can also access it since it already depends on the Card module. Otherwise we lose the consistency and this might go unnoticed later.